cns/restserver/ipam.go

```diff
@@ -34,6 +35,19 @@ const (
 	InterfaceName     = "eth0"
 )
 
+func checkNCStatuses(ncStatuses map[string]containerstatus) (string, bool) {
```

Comment:
I feel like this should be a type... something like:

```go
type multicontainerstatus map[string]containerstatus
```

The `string` returned from this can be implemented by `(*multicontainerstatus).String()`, and the `bool` can be given meaning by implementing a `(*multicontainerstatus).AnyFailed()` method or something.

Reply:
Updated